website/docs/reference/integrations/jira-cloud-plugin-usage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Jira Cloud Integration - Usage
+---
+
+With the Unleash Jira Cloud Plugin you can create, view and manage Unleash feature toggles directly from a Jira issue.
+
+The plugin also shows you current status of connected toggles.
+
+## Using the plugin
+
+To use the plugin, you'll need to create an issue or use an existing one. Once the issue is saved and you open up the issue panel, you'll be greeted with a button to activate the Unleash plugin for that issue.
+
+![Jira Cloud - Button to activate Unleash plugin in Jira Issue](/img/jira-cloud-activate-unleash-panel-button.png)
+
+### Connecting a toggle to an issue
+
+![Jira Cloud - Button that opens modal to connect toggles to Jira Issue](/img/jira-cloud-issue-button.png)
```

Comment:
What is this an image of? Is this what happens after you "activate the Unleash plugin" for this issue? or is this something else?

Is this correct?

```suggestion
<Figure caption="Once active, the Unleash plugin adds a button labeled 'connect toggle' to the issue." img="/img/jira-cloud-issue-button.png" />

```

Reply:
Yeah that seems to be a correct understanding. Activating unleash for the issue adds an issue panel section that at first will only contain that button